src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -980,11 +1204,12 @@ private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue cons
                 int tempsNeeded = MergeArgumentsAndSideEffects(arguments, storesToTemps);
                 if (tempsNeeded > 0)
                 {
-                    foreach (BoundAssignmentOperator s in storesToTemps)
+                    foreach (BoundExpression s in storesToTemps)
                     {
-                        if (s != null)
+                        Debug.Assert(s is null or BoundAssignmentOperator);
```

Comment:
>s is null  [](http://example.com/codeflow?start=37&length=10)

How did we end up with nulls in the array?

Reply:
`MergeArgumentsAndSideEffects` sets the position to `null` when it eliminates a store.